jest.polyfills.js

```diff
@@ -0,0 +1,32 @@
+// jest.polyfills.js
```

Comment:
These are required by Mock Service Worker to play nicely with Jest, assuming we wouldn't have these in the frontend repo.

https://mswjs.io/docs/migrations/1.x-to-2.x/#requestresponsetextencoder-is-not-defined-jest